os_stub/cryptlib_mbedtls/sys_call/timer_wrapper_host.c

```diff
@@ -15,13 +16,17 @@
 struct tm *mbedtls_platform_gmtime_r(const mbedtls_time_t *tt,
                                      struct tm *tm_buf)
 {
-    struct tm *lt;
-
-    lt = gmtime(tt);
+#ifdef _MSC_VER
+    if (gmtime_s(tm_buf, tt) != 0) {
```

Comment:
Should be `gmtime_s(tm_buf, tt) == NULL`, like below.

Reply:
> Should be `gmtime_s(tm_buf, tt) == NULL`, like below.
errno_t gmtime_s(
   struct tm* tmDest,
   const __time_t* sourceTime
);
typedef int                           errno_t;
C4047: "errno_t" and "void *" have different indirection levels

basis:
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/gmtime-s-gmtime32-s-gmtime64-s?view=msvc-170

Reply:
C11 defines it as `struct tm *gmtime_s(const time_t * restrict timer, struct tm * restrict result);` so I guess this compiler isn't following the standard? I guess in the short-term we'll go with the compiler.